tools/RealmBrowser/RealmBrowser/Classes/RLMDocument.m

```diff
@@ -55,6 +61,12 @@ - (instancetype)initWithContentsOfURL:(NSURL *)absoluteURL ofType:(NSString *)ty
             if ([realmNode connect:&error]) {
                 ws.presentedRealm  = realmNode;
 
+                ws.changeNotificationToken = [realmNode.realm addNotificationBlock:^(NSString *notification, RLMRealm *realm) {
+                    for (RLMRealmBrowserWindowController *windowController in ws.windowControllers) {
```

Comment:
It should be possible to reload only the window for the realm that changed here.